ludwig/constants.py

```diff
@@ -205,6 +206,34 @@
     "num_filter_bands",
 ]
 
+INPUT_FEATURE_KEYS = [
```

Comment:
AFAICT these constants are only used in `backward_compatibility.py`, maybe it would be simpler to define them there.

Reply:
Ya that's probably a good call. Will move them now.